src/sage/rings/invariants/invariant_theory.py

```diff
@@ -2526,9 +2526,9 @@ def _covariant_conic(A_scaled_coeffs, B_scaled_coeffs, monomials):
         (b0*c1+c0*b1-2*f0*f1) * monomials[0] +
         (a0*c1+c0*a1-2*g0*g1) * monomials[1] +
         (a0*b1+b0*a1-2*h0*h1) * monomials[2] +
-        2*(f0*g1+g0*f1 -c0*h1-h0*c1) * monomials[3] +
-        2*(h0*f1+f0*h1 -b0*g1-g0*b1) * monomials[4] +
-        2*(g0*h1+h0*g1 -a0*f1-f0*a1) * monomials[5]  )
+        2*(f0*g1+g0*f1 - c0*h1-h0*c1) * monomials[3] +
+        2*(h0*f1+f0*h1 - b0*g1-g0*b1) * monomials[4] +
+        2*(g0*h1+h0*g1 - a0*f1-f0*a1) * monomials[5]  )
 
```

Comment:
Why not for other `+` and `-`?

Reply:
this is automated using autopep8, fixing when there is space only on one side.

Reply:
By the general rule, there should be spaces around other pluses and minuses. I wonder if that would break the author's intention.

Reply:
If not adding spaces everywhere, removing spaces before the minus seems to make it more consistent.

Reply:
My main  goal here is to fix spaces around modulo (E228). The script autopep8 fails to do that correctly, only adding space on one side of `%`. That's the reason why I have to useautopep8 once again to fix E225, so that modulo gets a space on both sides. The lack of consistency does not get worse than it was. I think we can accept these minor side effects.

Reply:
OK if you do not intend to improve it more while we are at it.